### DISCUSSION TITLE
Cant transition the cart when one of the product got deleted from catalog

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1272] [Q&A]
I accidentally deleted one of the Products. Orders that are having this product have no way to cancel from Admin.

**Comment by michaelbromley** [comment]
You mean you want to cancel that deleted product from the order but you cannot?

**Reply by ramkishore-yerubandi** [reply]
The problem is when that deleted item is in active cart, from admin UI i cant cancel cart

**Reply by michaelbromley** [reply]
OK, can you create a bug report for this? Include steps to reproduce. Thanks!
